src/main/java/net/glowstone/block/blocktype/BlockType.java

```diff
@@ -419,4 +419,23 @@ public void onRedstoneUpdate(GlowBlock block) {
     public void onEntityStep(GlowBlock block, LivingEntity entity) {
         // do nothing
     }
+
+    /**
+     * The rate at which the block should be pulsed.
+     *
+     * @return null if the block should not pulse, or a number of ticks between pulses.
+     */
+    public Integer getPulseTickSpeed(GlowBlock block) {
```

Comment:
I chose Integer because you can return null. It's a lazy way of doing it; it checks if getPulseTickSpeed is null, then it doesn't require a pulse at all. Overriding and returning a value will give the tick delay.